services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx

```diff
@@ -57,50 +56,47 @@ const TopNavView = ({ isAdmin, loading, displaySearch, displayLogo, intl }) => {
     <Menu
       // Ant-design issue: recognizes either depending on machine
       // "-1" is used for off-screen content that appears on a specific event
-      /* eslint-disable react/jsx-no-duplicate-props */
       tabIndex={-1}
-      tabindex="-1"
```

Comment:
this was added by suku and benoit for accessibility reasons. are we sure we want to remove it? im ok with removing but once again @Rizvida1 should review this

Reply:
It works as it should